daily_open_prs.rb

```diff
@@ -1,99 +1,83 @@
 #!/usr/bin/env ruby
-#This script is to output a csv file to tally how many PRs are currently open on each day between two date ranges. It is also split into both PRs raised by community members and PRs raised by Puppet members.
+# frozen_string_literal: true
+
+# This script is to output a csv file to tally how many PRs are currently open on each day between two date ranges. It is also split into both PRs raised by community members and PRs raised by Puppet members.
 
 require 'optparse'
 require 'csv'
 require 'octokit'
 require_relative 'octokit_utils'
+require 'json'
+
+output = File.read('modules.json')
+parsed = JSON.parse(output)
 
 options = {}
 options[:oauth] = ENV['GITHUB_COMMUNITY_TOKEN'] if ENV['GITHUB_COMMUNITY_TOKEN']
 parser = OptionParser.new do |opts|
   opts.banner = 'Usage: open_and_created.rb [options]'
-
-  opts.on('-n', '--namespace NAME', 'GitHub namespace. Required.') { |v| options[:namespace] = v }
-  opts.on('-r', '--repo-regex REGEX', 'Repository regex') { |v| options[:repo_regex] = v }
   opts.on('-t', '--oauth-token TOKEN', 'OAuth token. Required.') { |v| options[:oauth] = v }
-  opts.on('-o', '--overview', 'Output overview, summary totals to csv') { options[:display_overview] = true}
-
-  # default filters
-  opts.on('--puppetlabs', 'Select Puppet Labs\' modules') {
-    options[:namespace] = 'puppetlabs'
-    options[:repo_regex] = '^puppetlabs-'
-  }
-  opts.on('--puppetlabs-supported', 'Select only Puppet Labs\' supported modules') {
-    options[:namespace] = 'puppetlabs'
-    options[:repo_regex] = OctokitUtils::SUPPORTED_MODULES_REGEX
-  }
-  opts.on('--community', 'Select community modules') {
```

Comment:
Hi @daianamezdrea Vox Pupuli still uses this. Is it possible to keep this option?